### ISSUE TITLE
Generate 6+ graphs with vertices skipped

### ISSUE THREAD

**Issue by MuellMark** [MuellMark/DDMGraphLabelingGenerator#17]
This will be worked on after #16, but eventually I'll have to alter the code to allow for 1 vertex to be all zeroes. Ideally I'll do this in a way that can be easily changed, since the same thing will have to be done for 7's

**Comment by MuellMark** [comment]
I'm going to open this issue up to graphs of size 6+, not just 6 graphs since the code should be the similar, if not the same across all graphs. It's not clear if this is something that will still be implemented, I'll update as needed

**Comment by MuellMark** [comment]
By accident I found a very easy way to do this so I made it it's own method. If this is desired I can outpoint everything. I'll keep this issue open until a verdict is received

**Comment by MuellMark** [comment]
All of these graphs will be generated by the array edge implementation since it's the fastest, and it already contains the method for it. I'll add them to the visualizations and make sure they're all organized before this is then closed. Hopefully 8 graphs will work, as there will likely be a lot of them.

**Comment by MuellMark** [comment]
I just figured out that the shapes for 7's are a bit off so I'm going to rerun the other 7's to get the better shape in, and look at the other shapes to avoid this problem

**Comment by MuellMark** [comment]
All of the graphs were generated and they are in their own folders. If there is a desire I can generate all of the graphs with a skipped edge in their own folder, but I'll make a new issue if this is desired.
